client/components/language-picker/modal.jsx

```diff
@@ -140,12 +177,32 @@ class LanguagePickerModal extends PureComponent {
 		);
 	};
 
+	renderSuggestedLanguages() {
+		const { suggestedLanguages } = this.state;
+		if ( ! suggestedLanguages ) {
+			return null;
+		}
+
+		return (
+			<div className="language-picker__modal-suggested">
+				<div className="language-picker__modal-suggested-inner">
+					<div className="language-picker__modal-suggested-label">
+						{ this.props.translate( 'Suggested languages: ' ) }
```

Comment:
The trailing space is an indicator of string concattenation. Can we add the languages as a component in the translation? If not, please remove the whitespace and the translators will accommodate as good as possible.

Reply:
Unfortunately there's a lot of cruft around the language texts. I'll remove the white space for now.